src/Sarif.Driver/Sdk/PluginDriverCommand.cs

```diff
@@ -37,6 +42,102 @@
             return assemblies;
         }
 
+        public bool ValidateExportUri(string outputFilePath, string exportUri)
```

Comment:
>ValidateExportUri [](http://example.com/codeflow?start=20&length=17)

This isn't a good public helper. Why would our plug-in driver class has a simple URI validation API that it exposes to all classes?

Reply:
thanks for the suggestion, when i first created it, i followed all the other methods (all were public).

Now, i'm changing to internal or internal static.

thanks for the suggestions :)